GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
+## Role
+  * Group of permissions
+  * Can be attached to a group
+## Rule
+## Tag
```

Comment:
Tag - **Not currently implemented** The idea is for Relay to have a name, but can have multiple tags associated with it. This tag can be used when defining rules for certain bundled commands. Tags will be immutable. This reduces the complexity of needing to update rules with certain tags when a tag is changed for a certain relay or group of relays.